examples/nodejs/src/index.ts

```
@@ -66,3 +80,29 @@ const privateKey = 'xxx'
+    }),
+  })
+
+  const data = await sdk.getOrder({
+    chainId: SupportedChainId.MAINNET,
+    env: 'staging',
+    rpcProvider: new JsonRpcProvider('https://mainnet.gateway.tenderly.co'),
+    orderId:
+      '0xb8aeae0626654ea134614a42044dcf081544981e19f35082e20c967d5210834dfb3c7eb936caa12b5a884d612393969a557d430768248d30',
+  })
+
+  console.log('Bidging order data', data)
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)